docs/how_to/aggregator.md

```diff
@@ -0,0 +1,159 @@
+# Setting up m3aggregator
+
+## Introduction
+
+m3aggregator is used to cluster stateful downsampling and rollup of metrics before they are store in M3DB. The m3coordinator also performs this role but is not cluster aware. This means metrics will not get aggregated properly if you send metrics in round round fashion to multiple m3coordinators for the same metrics ingestion source (e.g. Prometheus server).
+
+Metrics sent to m3aggregator are correctly routed to the instance resposible for aggregating each metric and there is multiple m3aggregator replicas to ensure no single point of failure during aggregation.
+
+## Configuration
+
+Before setting up m3aggregator, make sure that you have at least [one M3DB node running](single_node.md) and a dedicated m3coordinator setup.
+
+We highly recommend running with at least a replication factor 2 for a m3aggregator deployment.
+
+### Running
+
+You can run m3aggregator by either building and running the binary yourself:
+
+```
+make m3aggregator
+./bin/m3aggregator -f ./src/aggregator/config/m3aggregator.yml
+```
+
+Or you can run it with Docker using the Docker file located at `$GOPATH/src/github.com/m3db/m3/docker/m3aggregator/Dockerfile`.
+
+### Coordinator configuration
+
+Metrics will still arrive at the m3coordinator, they simply need to be forwarded to m3aggregators. The m3coordinator then also needs to receive metrics that have been aggregated from the m3aggregator and store them in M3DB, so running an ingestion server should be configured.
```

Comment:
a digram here would be super helpful since its pretty confusing to have this prometheus --> m3coordinator --> m3aggregator --> m3coordinator flow.

Would also be good to say that the m3coordinators in that diagram can be different